README.md

```diff
@@ -53,6 +53,7 @@ const IGNORE_FOLDERS = ['.git', 'node_modules'];
 
 module.exports = {
   debug: false,           // changes the output to show debug information, cmd and stdout output
+  delete: true,           // delete extraneous files from destination
```

Comment:
How would you feel about calling this `rsyncDelete`  or `syncDelete`?  I am worried that `delete` may be too vague in the config file.

Reply:
Absolutely, do you have a preference between those two?

Reply:
I think I like `syncDelete` it's generic, has nothing to do with rsync and could work for other synchronization tools, if I ever make that an option.